fbcrawl/settings.py

```diff
@@ -93,3 +103,5 @@
 DUPEFILTER_DEBUG = True
 LOG_LEVEL = 'INFO'
 #LOG_LEVEL = 'DEBUG'
+
+IMAGES_STORE = '/Users/theblackcat/Desktop/fbcrawl/images/'
```

Comment:
Why was this left unchanged in the repository? Who wants to know where he saved photos on in what appears to be a windows desktop. Ah, yuk...windows...makes me want to shower....